docs/index.restdown

```diff
@@ -1081,6 +1082,21 @@ Options:
 
 Note that `ip`, `xff` and `username` are XOR'd.
 
+### Request Expiry
+
+Request expiry is used to kick out old requests that have been timed out somewhere upstream.
```

Comment:
I might change the prose here to something a bit more succinct, like.

"Request Expiry can be used to throttle requests that have already exceeded their timeouts. Requests can be sent with a configured header, e.g. 'x-request-expiry-time', which gives in absolute wall time, when this request will be timed out by the client. 

This plugin will throttle all incoming requests via a 504 where 'x-request-expiry-time'  < Date.now() -- since these incoming requests will no longer be processed by the client."